src/index.html

```diff
@@ -11,7 +11,145 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="regBlock">
+        <legend>Personal information</legend>
+        <div class="form">
+          <label>
+            Surname:
+            <input type="text"
+             name="surname"
+             autocomplete="off">
+          </label>
+        </div>
+        <div class="form">
+          <label>
+            Name:
+            <input type="text"
+             name="name"
+             autocomplete="off">
+          </label>
+        </div>
+        <div class="form">
+          <label>
+          How old are You?
+            <input
+           name="age"
+           type="number"
+           value="12"
+           min="1"
+           max="100">
```

Comment:
```suggestion
            <input
             name="age"
             type="number"
             value="12"
             min="1"
             max="100"
           >
```
Проверь форматирование своего кода(смотри Good Example)
<img width="1178" alt="formatting" src="https://user-images.githubusercontent.com/94292654/180417060-b8a68739-2586-4ab0-9a41-6da0b1f1290d.png">